platform/cli/usage-guides/new-service/java/aws/README.md

```diff
@@ -0,0 +1,344 @@
+  <img src="./pictures/launch-terragrunt-pipeline-apply-application-output-01.png" /><br>
+  output truncated... <br>
+  <img src="./pictures/launch-terragrunt-pipeline-apply-application-output-02.png" />
+</p>
+
+Deploy the webhooks for the Java application  service
+```
+~ $ launch terragrunt --target-environment root --platform-resource webhook --apply --generation
+```
+<p align="center">
+  <img src="./pictures/launch-terragrunt-webhook-apply-application-output-01.png" /><br>
+  output truncated... <br>
+  <img src="./pictures/launch-terragrunt-webhook-apply-application-output-02.png" />
+</p>
+
+Deploy the Java application service
+```
+~ $ launch terragrunt --target-environment sandbox --platform-resource service --apply --generation --render-app-vars
+```
+<p align="center">
+  <img src="./pictures/launch-terragrunt-service-apply-application-output-01.png" /><br>
+  output truncated... <br>
+  <img src="./pictures/launch-terragrunt-service-apply-application-output-02.png" />
+</p>
+
+#### 4.1.6 Connect the webhooks
+Within this section, we need to connect the webhooks for the newly created service and the Java application. Both of these repositories will utilize the same webhooks.
+
+In this first section, we will connect the new created repository.
+- In Github, navigate to the newly created repository. Under `Settings`, navigate to `Webhooks` and click `Add webhook`.
+<p align="center">
+  <img src="./pictures/github-settings-webhook-app-props.png" /> 
+</p>
+
+We are going to create webhooks for each of the lambdas (4) for the properties repo.
+- In `Payload URL` put the function URL of the lambda
+- In `Content type`, select `application/json`
+- In `Secret`, paste your git_lambda_secret created earlier.
+- Select `Let me select individual events`
+  - Only select `Pull requests`
+
+<p align="center">
+  <img src="./pictures/github-settings-webhook-create-platform-01.png" /><br>
+  <img src="./pictures/github-settings-webhook-create-platform-02.png" />
+</p>
+
+
+We will now connect the webhooks for the Java application
+- In Github, navigate to the Java application repository. Under `Settings`, navigate to `Webhooks` and click `Add webhook`.
+<p align="center">
+  <img src="./pictures/github-settings-webhook-app-java.png" /> 
+</p>
+
+We are going to create webhooks for each of the lambdas (4) for the Java Application. 
+- In `Payload URL` put the function URL of the lambda
+- In `Content type`, select `application/json`
+- In `Secret`, paste your git_lambda_secret created earlier.
+- Select `Let me select individual events`
+  - Only select `Pull requests`
+
+<p align="center">
+  <img src="./pictures/github-settings-webhook-create-platform-01.png" /><br>
+  <img src="./pictures/github-settings-webhook-create-platform-02.png" />
+</p>
+
+## 5. **View the Java Application**
+
+Perform this guide to deploy a AWS Client VPN utilizing the launch platform to view this application in a private VPC:
+
+- [Deploy Client VPN]()
```

Comment:
Broken link.

Reply:
https://dev.azure.com/launch-dso/platform-accelerators/_workitems/edit/141 created for this documentation effort, will be working on it starting tomorrow.